franklin/config/settings/base.py

```diff
@@ -200,19 +200,26 @@
             'backupCount': LOGFILE_COUNT,
             'formatter': 'simple'
         },
+        'paper_trail': {
+            'level': 'DEBUG',
+            'filters': ['require_debug_false'],
+            'class': 'logging.handlers.SysLogHandler',
+            'formatter': 'verbose',
+            'address': ('logs.papertrailapp.com', 63032)
+        }
```

Comment:
I don't think we need to worry about that for now. There are some ways we can lock it down, but I'll hop on that next sprint.